libvast/vast/format/reader.hpp

```diff
@@ -29,6 +31,14 @@ class reader {
 public:
   // -- member types -----------------------------------------------------------
 
+  enum class inputs { file, inet };
```

Comment:
How about using the two categories `file` and `net`. The `i` in `inet` suggests "Internet" to me. I think all we want is to distinguish between file and network-based input.

Reply:
The suggestive nature is intentional, I wanted to convey that this can be used with ipv4/6. This naming is pretty prevalent in posix APIs (e.g `AF_INET`). Another value we might add here in the far future is `zigbee` which is also a network, but does not use the same addressing scheme.

Reply:
Convinced. I'm fine with calling it `inet` now.